core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -887,7 +887,8 @@ default Optional<ConnectorOutputMetadata> finishRefreshMaterializedView(
             ConnectorInsertTableHandle insertHandle,
             Collection<Slice> fragments,
             Collection<ComputedStatistics> computedStatistics,
-            List<ConnectorTableHandle> sourceTableHandles)
+            List<ConnectorTableHandle> sourceTableHandles,
+            List<String> sourceTableFunctions)
```

Comment:
No requested change. A function contains the schema name. We will need to re-modify this method if we want to change the subsequent behavior based on the function.

Reply:
yes, but that is probably a distant future